Gold standard/Sorting and searching/binarySearch.js

```diff
@@ -0,0 +1,49 @@
+var MIN_INT_NUMBER = 0;
+var MAX_INT_NUMBER = 10;
+var ARRAY_LENGTH = 10;
+
+function getRandomInt(min, max) {
+  	return Math.floor(Math.random() * (max - min)) + min;
+}
+
+function createSortedIntNumbersArrayByLength(length){
+	var array = [];
+	var i = 0;
+	
+	while(i < length){
+		array.push(getRandomInt(MIN_INT_NUMBER, MAX_INT_NUMBER));
+		i++;
+	}
+
+	return array.sort(function(a, b){ return a - b; });
+}
+
+Array.prototype.recursiveBinarySearch = function(num){
+	var start = 0;
```

Comment:
Why does function have name recursiveBinarySearch? It doesn't seem to be recursive